ang/civicase/CaseCard--case-list.html

```diff
@@ -4,15 +4,15 @@
   * use 'civicase__animated-checkbox-card--expanded' class : for enabled checkbox state
   * Update links with activityFeedUrl function activityFeedUrl(caseid, activityType to link, activityStatus to Link)
   -->
-<div class="panel panel-default panel-strip-primary civicase__case-card civicase__animated-checkbox-card" ng-style="{'border-left-color': data.color}" ng-class="{'civicase__case-card--active': $parent.viewingCase === data.id, 'civicase__case-card--closed': data.status === 'Resolved'}" >
+<div class="panel panel-default panel-strip-primary civicase__case-card civicase__animated-checkbox-card" ng-style="{'border-left-color': data.color}" ng-class="{'civicase__case-card--active': $parent.viewingCase === data.id, 'civicase__case-card--closed': data.status === 'Resolved', 'civicase__animated-checkbox-card--expanded': $parent.showCheckboxes && !$parent.viewingCase && $parent.selectedCases && $parent.bulkAllowed}" >
```

Comment:
We should need to mention this many conditions.
`$parent.showCheckboxes && !$parent.viewingCase && $parent.selectedCases && $parent.bulkAllowed`

When `bulkAllowed` is false, `showCheckboxes` should be disabled by default. In fact the Bulk Allowed logic should be part of BulkActions directive. And unless user have permission, showCheckboxes should be disabled by default.

Also could you remind me, why `!$parent.viewingCase && $parent.selectedCases` is needed to be checked here?

Reply:
> When bulkAllowed is false, showCheckboxes should be disabled by default. In fact the Bulk Allowed logic should be part of BulkActions directive. And unless user have permission, showCheckboxes should be disabled by default.

`bulkAllowed` is for permissions and show checkbox is for toggle view, both are different. However, this is a nice idea to make BulkActions to be a part of the directive. Let me see to this. and get back to you.

> Also could you remind me, why !$parent.viewingCase && $parent.selectedCases is needed to be checked here?

Sure. In the current design the checkbox doesn't show up when any case is in view state also if at least one case is selected, that means checkbox were enabled and if user navigates within pagers then the checkboxes should be visible. Making user know that he/she have some selection already in progress. Also if one of the cases is selected the actions dropdown will be available and if it is then the user should see the checkboxes for each cases throughout pagers.

Reply:
Lets discuss this.